docs/contributors_guide/Contribution_Procedure.rst

```diff
@@ -77,15 +80,125 @@ The first commit message should be the following format:
       after the issue is merged
 
+.. _gsg-contributors_guide-contribution_procedure-testing_your_submissions:
+
+Testing Your Submissions
+------------------------
+
+First off, thank you again for your contribution! Things don't get better
+without your help!
+
+This section contains two sets of guidelines. First, ones that are recommended
+for external contributions from the community. Second, ones that are expected
+to be adhered to by the core development team.
+
+External Contributors
+^^^^^^^^^^^^^^^^^^^^^
+
+We will happily accept all levels of contributions, small, medium, or large
+without any tests.
+
+However, for us to quickly and effectively assess your contribution you should
+either add unit (``rspec-puppet``) and/or acceptance (``beaker``) tests.
+
+As the size of the contribution increases, this becomes increasingly important,
+because, depending upon the complexity of the changes, it may simply be too
+difficult to do a timely assessment of such a contribution without corresponding
+tests. In these cases, it would be best if you split your contribution into
+smaller pull requests that are easier to assess.
+
+Core Developer Contributions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The core development team is expected to follow these guidelines when adding
+code to the project.
+
+In all cases, a cursory ``grep`` through the `simp-doc`_ project should be done
+and a ticket should be entered if the overall project documentation may be
+affected by your change.
+
+This should also be done by the core development team for any external
+contributions, since it is unreasonable to expect external contributors
+to take the effort to dig through the `simp-doc`_ project.
+
+Trivial Contributions
+"""""""""""""""""""""
+
+Trivial contributions are those that constitute a small documentation update,
+code correction, or bug fix consisting of only a few lines. These contributions
+must not negatively impact the behavior of the user experience or code.
+
+Trivial contributions do not require an associated ticket and may be covered
+under a ``SIMP-MAINT`` branch.
+
+Trivial contributions require **one maintenance team member** review and may
+optionally add additional unit or acceptance testing.
+
+Minor Contributions
+"""""""""""""""""""
+
+Minor contributions are those that add a feature or fix a larger bug in
+component that are more than five or ten lines and/or are not only
+documentation updates.
+
+Minor contributions must have unit tests and should have acceptance tests.
+Acceptance tests may be deferred but a ticket must be filed with an explanation
+and a link in the PR if the acceptance test addition is deferred.
+
+Minor contributions require **one maintenance team member** review. The
+reviewing team member may decide that acceptance tests are required based on
+the understandability of the contribution.
+
+Major Contributions
+"""""""""""""""""""
+
+Major contributions are any changes that affect multiple parts of the system,
```

Comment:
On my previous team, two-person reviews were also required when the risk of the solution being correct was high, because of the inexperience of the contributor  WRT the subject matter.  This provided both a sanity check and a mechanism to bring folks up to speed in areas new to them.